client/src/main/python/slipstream/cloudconnectors/BaseCloudConnector.py

```diff
@@ -726,6 +726,12 @@ def __get_obfuscation_script(self, orchestrator_public_key, username, user_info=
         command += "[ -x /etc/init.d/sshd ] && { service sshd reload; } || { service ssh reload; }\n"
         return command
 
+    def _get_bootstrap_script_if_not_build_image(self, node_instance, pre_export=None,
+                                                 pre_bootstrap=None, post_bootstrap=None,
+                                                 username=None):
+        return self._get_bootstrap_script(node_instance, pre_export, pre_bootstrap, post_bootstrap, username) \
+            if not self.is_build_image() else ''
+
```

Comment:
We now probably have to change all the connectors to use `_get_bootstrap_script_if_not_build_image`, while allowing them to override `_get_bootstrap_script`.  Right?